README.md

```diff
@@ -1,2 +1,175 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://websitesetup.org/html-tutorial-beginners/">Html</a></li>
+    <li><a href="https://www.codecademy.com/learn/learn-css">Css</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">None</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="#">None</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+
+
+### Key Features <a name="key-features"></a>
+- **accessible heading**
+- **color full heading**
+- **smoth lokking**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://github.com/Paul-tes/Hello-Microverse)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+In order to run this project you need:
+- install web broweser
+### Setup
+Clone this repository to your desired folder:
+- git clone https://github.com/Paul-tes/Hello-Microverse.git
+### Install
+Install this project with:
+### Usage
+
+To run the project, execute the following command:
+- open index.html with your browser that found in Hello-Microverse folder 
+
+### Run tests
+
+To run tests, run the following command:
+- right Click + open With + chrome
+### Deployment
+You can deploy this project using:
+- N/A
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+👤 **Pawlos Tesfaye**
+
+- GitHub: [@paul-tes](https://github.com/paul-tes)
+- LinkedIn: [paul](https://www.linkedin.com/in/paul-tesfaye-687820215/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+- [ ] **[Provide Live Demo]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project make sure to give it a star 🌟
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+I would like to thank Microverse Team. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- LICENSE -->
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] The license link shows a 404 error. Kindly add the MIT license to your repo.  This is so your Readme will be professional. A professional readme file shouldn't have broken links. You can get the MIT license file from [this repo.](https://github.com/microverseinc/readme-template)